LayoutTests/fast/html/disconnected-html-plugin.html

```diff
@@ -0,0 +1,65 @@
+<iframe
+    srcdoc='<html>
+        <script>
+        function gc() {
+         {
+         }
+        }
```

Comment:
This `gc()` function does literally nothing.  :)

There's a `gc()` function in `LayoutTests/resources/js-test-pre.js` that should probably be used instead, unless this one can just be deleted.

Have you tried reducing the test case by hand?

Reply:
Nisha told me she tried to remove individual try/catch lines, but couldn't remove any single line without making the test case stop working.

Reply:
> This `gc()` function does literally nothing. :)
> 
> There's a `gc()` function in `LayoutTests/resources/js-test-pre.js` that should probably be used instead, unless this one can just be deleted.
> 
> Have you tried reducing the test case by hand?

Able to remove this gc() function successfully.

Reply:
> Nisha told me she tried to remove individual try/catch lines, but couldn't remove any single line without making the test case stop working.

Yes I tried removing try() / catch() blocks one at a time but can't reduce further